apollo-backend/src/main/java/io/logz/apollo/blockers/types/GHCommitStatusBlocker.java

```diff
@@ -0,0 +1,28 @@
+package io.logz.apollo.blockers.types;
+
+import io.logz.apollo.blockers.BlockerFunction;
+import io.logz.apollo.blockers.BlockerInjectableCommons;
+import io.logz.apollo.blockers.BlockerType;
+import io.logz.apollo.models.DeployableVersion;
+import io.logz.apollo.models.Deployment;
+import io.logz.apollo.scm.GithubConnector;
+import java.io.IOException;
+
+@BlockerType(name = "githubCommitStatus")
+public class GHCommitStatusBlocker implements BlockerFunction{
+
+    @Override
+    public void init(String jsonConfiguration) throws IOException {
+
```

Comment:
Why is it needed?

Reply:
it required to implement the `BlockerFunction` interface